src/cli/test.py

```diff
@@ -80,7 +100,7 @@ def print_text(line_number, text):
 
 # Wrap everything in an keyboard interupt handler
 try:
-	while True:
+	while cv2.getWindowProperty("Howdy Test", cv2.WND_PROP_VISIBLE) > 0:
```

Comment:
This looks really nice, will this properly close the window when you click the X?

Reply:
Yes, I found that in other test for opencv and I find it really useful and intuitive